frontend/kesaseteli/employer/src/pages/application.tsx

```diff
@@ -0,0 +1,41 @@
+import CompanyInfo from 'kesaseteli/employer/components/companyInfo/CompanyInfo';
+import EmployerForm from 'kesaseteli/employer/components/employerForm/EmployerForm';
+import withAuth from 'kesaseteli/employer/hocs/withAuth';
+import { GetStaticProps, NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { useTranslation } from 'next-i18next';
+import * as React from 'react';
+import FormSection from 'shared/components/forms/section/FormSection';
+import Layout from 'shared/components/Layout';
+import getServerSideTranslations from 'shared/i18n/get-server-side-translations';
+import { DEFAULT_LANGUAGE } from 'shared/i18n/i18n';
+import { getFirstValue } from 'shared/utils/array.utils';
+
+const ApplicationPage: NextPage = () => {
+  const { t } = useTranslation();
+  const router = useRouter();
+
+  const locale = router.locale ?? DEFAULT_LANGUAGE;
+
+  const applicationId = getFirstValue(router.query.id);
+
+  if (!applicationId) {
+    void router.push(`${locale}/`);
```

Comment:
404 error message is shown if id exists on url but not in database. for example `/application?id=fooBar` would show it. 
But if the id-param is missing then it redirects: `/application?foo=bar` => `/`
Also, the index page works now so that it always automativally either load the draft application, or create new one if it does not exist. So in this case it would redirect twice: `/application?foo=bar` => `/` => `/application?id=correctId`